### PR TITLE
Add release notes for 0.9.6

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: website/blog/2020-11-20-lithium.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.6 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.9.6, which mostly concentrates
+on adding support for the newly released Scala 2.13.4, but also adds some
+improvements for alreadye xisting features.
```
Comment (ayoub-benali) [inline website/blog/2020-11-20-lithium.md:10]:
```suggestion
improvements for already existing features.
```

Comment (tgodzik) [reply website/blog/2020-11-20-lithium.md]: Tnaks! Fixed! Should have turned on the spellchecker before :sweat_smile:

File: website/blog/2020-11-20-lithium.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+## Show implicit conversions and classes
+
+Last
+[release](/metals/blog/2020/11/10/lithium.html#show-implicits-and-type-decorations)
+introduced the option to display additional data inferred from the code to a
+user. Starting with this release it's also possible to show implicit conversions
+and classes, which can be enabled by the additional
+`showImplicitConversionsAndClasses` setting. Moreover, Visual Studio Code
+extensions has 3 new commands to toggle all these new options, which enables
```
Comment (ckipp01) [inline website/blog/2020-11-20-lithium.md:62]:
```suggestion
`showImplicitConversionsAndClasses` setting. Moreover, the Visual Studio Code
extension has 3 new commands to toggle all these new options, which enables
```

File: website/blog/2020-11-20-lithium.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+Previously, it was possible to navigate a stacktrace using the
+`Analyze stacktrace` command which was added in the
+[v0.9.4](/metals/blog/2020/09/21/lithium.html#analyze-stacktrace-command)
+release. It turns out, we can reuse the same mechanism to show file links in the
+`Debug Console` in Visual Studio code:
+
+![navigate-stacktrace](https://i.imgur.com/qeitymN.gif)
+
+This was achieved by adding additional information to the output already sent to
+the editor, so this additional file links should also be reused by an other DAP
```
Comment (ckipp01) [inline website/blog/2020-11-20-lithium.md:84]:
```suggestion
the editor, so this additional file links should also be reused by any other DAP
```

**Review by ckipp01 — COMMENTED** [review]
Just a couple typos! Other than that, what do you think about mentioned the newly created https://scalameta.org/metals/docs/troubleshooting/faq.html page to help troubleshoot issues? It'd be great if people were super aware of it to both help of figure out what we should put on there and to also help answer some of the common question we always come across.

File: website/blog/2020-11-20-lithium.md (hunk 2 above)
Comment (tgodzik) [reply website/blog/2020-11-20-lithium.md]: Fixed!

File: website/blog/2020-11-20-lithium.md (hunk 3 above)
Comment (tgodzik) [reply website/blog/2020-11-20-lithium.md]: Fixed

**Comment by tgodzik** [comment]
> Just a couple typos! Other than that, what do you think about mentioned the newly created https://scalameta.org/metals/docs/troubleshooting/faq.html page to help troubleshoot issues? It'd be great if people were super aware of it to both help of figure out what we should put on there and to also help answer some of the common question we always come across.

I added a separate paragraph about, what do you think?

**Review by ckipp01 — APPROVED** [review]
Looks good! LGTM!

**Comment by tgodzik** [comment]
Starting the release!

**Pull request merged**